clvm-utils/src/lib.rs

```diff
@@ -1,5 +1,5 @@
+mod curry;
 mod tree_hash;
-mod uncurry;
```

Comment:
since you removed the readme, maybe you could document how to use the new uncurry mechanism in rustdoc here